proto/zitadel/auth.proto

```diff
@@ -869,7 +869,7 @@ service AuthService {
 
         option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
             tags: "User Memberships"
-            summary: "List My ZITADEL Permissions";
+            summary: "List My ZITADEL Manager Roles";
             description: "Show all the permissions my user has in ZITADEL (ZITADEL Manager)."
```

Comment:
In my opinion this is not correct, as this effectively does return the permissions.
Role: ORG_OWNER
Permission: org.read, user.write, user.read

Reply:
Ah I see, in that case I agree with you.

Reply:
I just realised that changing the summary doesn't make sense as well

Reply:
Sorry I went through it, and you are right. I got confused because we have two different, one that returns the roles and one that returns the permissions.